modules/chapter2/pages/section3.adoc

```diff
@@ -40,15 +22,25 @@ dr-xr-x---. 13 root root  4096 Aug 22 15:18 ..
 -rw-r--r--.  1 root root 12127 Aug 22 15:20 kubeconfig
 ----
 
+. Get the `kubeconfig` file, password for `kubeadmin` user and web console url from console.redhat.com.
```

Comment:
Get the 'kubeconfig; file, the password for 'kubeadmin user, and the web console URL from console.redhat.com.